sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.netstandard2.0.cs

```diff
@@ -862,20 +862,20 @@ public partial class CacheKeyQueryStringActionProperties
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct CacheLevel : System.IEquatable<Azure.ResourceManager.Cdn.Models.CacheLevel>
+    public readonly partial struct CdnCacheLevel : System.IEquatable<Azure.ResourceManager.Cdn.Models.CdnCacheLevel>
```

Comment:
Can we leave it as it is? Prepending the RP prefix here doesn't feel right. We can keep it as we follow the guide that first come first serve.

Reply:
This is cdn cache, so I think `Cdn` prefix is more accurate